pseudonymisation/3-la-pseudonymisation-en-pratique.md

```diff
@@ -82,4 +82,4 @@ Comme évoqué dans la partie précédente, il existe de nombreuses interfaces d
 
 ## Voir la pseudonymisation en action
 
-Vous pouvez directement explorer et réutiliser le code de notre outil sur [https://github.com/etalab-ia/pseudonymisation_decisions_ce](https://github.com/etalab-ia/pseudonymisation_decisions_ce).
+Vous pouvez directement explorer et réutiliser le code de notre outil sur [http://datascience-01.infra.data.gouv.fr/pseudo/](https://github.com/etalab-ia/pseudo_app).
```

Comment:
Est-ce qu'on peut avoir du HTTPS sur ce lien ? N'y a-t-il pas un sous-domaine pour cet outil ?

`datascience-01.infra.data.gouv.fr` ressemble à une adresse de machine et non d'une application

Reply:
Oui tout à fait elle est un peu brutale cette adresse, je vais voir avec @abulte pour avoir un subdomain. Je crois que pour l'instant on peut l'accepter car les anciens liens dans notre guide ne sont plus valides.

Reply:
Je fais ça de suite !

Reply:
Les urls en .infra sont à proscrire dans la nature

Reply:
Top! Merci!

Reply:
Je dois faire quelque chose de mon côté ?

Reply:
datascience.etalab.studio/pseudo/ devrait marcher tout de suite.

J'ai aussi ajouté *.datascience.etalab.studio vers la même machine, donc tu pourrais faire pseudo.datascience.etalab.studio, mais là y'a un peu de conf de ton côté en plus.

Pour le HTTPS/SSL ce serait pas mal, surtout qu'on parle de données à pseudonymiser 😬 https://certbot.eff.org for the win.

Reply:
Ok. Je vais corriger cela et je vais mettre le SSL. Ajouter une notice sur le traitement des données serait bien aussi :/  
(Elle est géniale l'interface du certbot... ça donne des idées pour le rendu Piaf)
Merci à vous deux!

Reply:
@psorianom Tu peux me re request en review quand tu aurais fais la conf de ton côté que je te mettes une ✅  et que je merge cette PR

Reply:
@AntoineAugusti  Oui, d'accord, je ferais ça.